### ISSUE TITLE
Bilinear Warping Compiling Issues 

### ISSUE THREAD

**Issue by sbhatia4** [D-Nilsson/GRFP#12]
Tensorflow - 1.5.0 
CUDA - 9.0 
Windows 10 
misc\compile.sh throws the below error, even though, I have bilinear_warping.cu.cc and .cc files in the folder. 

bilinear_warping.cu.cc
c1xx: fatal error C1083: Cannot open source file: 'bilinear_warping.cu.cc': No such file or directory
g++.exe: error: bilinear_warping.cc: No such file or directory
g++.exe: error: bilinear_warping.cu.o: No such file or directory

Any help would be highly appreciated!!

**Comment by LittleBear666666** [comment]
Hello. Have you solved the problem yet

**Comment by sbhatia4** [comment]
Hi, 
On Windows 10: 
Still, see the same issue.. 

On Linux: 
Tensorflow -  2.4
CUDA - 11.0 
Ubuntu 20.04 
I get the same error message, even though, I see .so file generated [so not sure, something wrong with scripts] but then, I got stuck with Tensorflow Contrib not available with Tensorflow 2.* when I run evaluate.py 

Can you please, tell the config, which you guys used at your end ? I will try with that config on Linux. 

And if you have some solution for the issue on Windows 10 system, would be great!! Thanks a lot!!

**Comment by burhanmudassar** [comment]
Are you running compile from within the misc directory?

**Comment by sbhatia4** [comment]
Yes, able to fix this issue on Linux.. with older version of CUDA [9.0] and tensorflow [1.9.0]. 

We can close this issue. Thanks a lot!
